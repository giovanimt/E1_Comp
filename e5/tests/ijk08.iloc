addI rsp, 4 => rsp
addI rsp, 4 => rsp
addI rsp, 4 => rsp
loadI 15 => r0
addI rfp, 4 => r1
store r0 => r1
loadI 3 => r2
addI rfp, 8 => r3
store r2 => r3
addI rfp, 4 => r4
load r4 => r5
addI rfp, 8 => r6
load r6 => r7
mult r5, r7 => r8
loadI 2 => r9
loadI 20 => r10
mult r9, r10 => r11
loadI 4 => r12
div r11, r12 => r13
add r8, r13 => r14
addI rfp, 0 => r15
store r14 => r15
