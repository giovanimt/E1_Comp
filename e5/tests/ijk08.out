00036178 55
00036182 15
00036186 3
